extensions/interactions/CodeRepl/CodeRepl.html

```diff
@@ -52,15 +52,19 @@
 <script type="text/ng-template" id="interaction/CodeRepl">
   <div class="code-repl-container">
     <div>
-      <div class="code-repl-input-box">
-        <div ui-codemirror="{onLoad: initCodeEditor}">
+      <!-- This ng-form is needed for submitting answer on hitting enter. -->
+      <form ng-submit="runAndSubmitCode(code)">
```

Comment:
This was there earlier (in the current code-base) so I changed it back to how it was. Should I remove it?